tests/test_cli_gui/test_search.py

```diff
@@ -131,6 +132,7 @@ def test_list_skills():
     _test_search_items_with_query("skill", "test")
 
 
+@pytest.mark.unstable
```

Comment:
This will stop the test from running entirely. Also, I never saw this test being unstable. Are you sure it was not failing previously because it was not extended...?

Reply:
I suggest removing the mark again